src/components/calendar/calendar.css

```diff
@@ -37,6 +38,19 @@
   padding-top: var(--spacing-medium);
 }
 
+.calendar-year-list {
+  height: calc(var(--spacing-x-large) * 7); /* 7 list options */
+  padding-left: var(--spacing-medium);
```

Comment:
Why the padding? It looks like we hav edouble padding and it adds odd whitespace to the left of each list item